src/gui/editors/AutomationEditor.cpp

```diff
@@ -654,6 +671,52 @@ void AutomationEditor::mousePressEvent( QMouseEvent* mouseEvent )
 				}
 			break;
 			}
+			case EditMode::EditTangents:
+			{
+				if (m_clip->canEditTangents())
+				{
+					m_clip->addJournalCheckPoint();
+
+					// Gets the closest node to the mouse click
+					timeMap::iterator node = getClosestNode(mouseEvent->x());
+
+					// Starts dragging a tangent
+					if (m_mouseDownLeft)
+					{
+						if (node != tm.end())
+						{
+							// Lock the tangents from that node, so it can only be
+							// manually edited
+							node.value().setLockedTangents(true);
+
+							m_draggedTangentTick = POS(node);
+							if (posTicks >= m_draggedTangentTick)
+							{
+								m_draggedOutTangent = true;
+							}
+							else
+							{
+								m_draggedOutTangent = false;
+							}
```

Comment:
Could be less verbose:
```suggestion
m_draggedOutTangent = posTicks >= m_draggedTangentTick;
```